starknet_py/tests/e2e/mock/compile_contracts.sh

```diff
@@ -1,32 +1,84 @@
 #!/bin/bash
+set -e
 
-MOCK_DIRECTORY=starknet_py/tests/e2e/mock
-CONTRACTS_DIRECTORY="$MOCK_DIRECTORY"/contracts
-CONTRACTS_COMPILED_DIRECTORY="$MOCK_DIRECTORY"/contracts_compiled
+MOCK_DIRECTORY="starknet_py/tests/e2e/mock"
```

Comment:
nitpick: this script will be only runnable from the repo root atm. Maybe we should do `MOCK_DIRECTORY="$(git rev-parse --show-toplevel)/starknet_py/tests/e2e/mock"` so it can be run from anywhere in the repo?